### PR TITLE
docs: tiny enhancements on documentation("getting-started", "plugin" )

### PR DESCRIPTION
**Pull request by piglei**
### What this PR does / why we need it:

- ("plugin.md") Fix a grammer error (zh language only)
- ("getting-started.md") Edit "Create a Route - How it works" by replacing the example `Route` payload with an actionable curl command to reduce cognitive overhead

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/en/latest/getting-started.md [hunk 1]
```diff
@@ -138,35 +138,36 @@ Apache APISIX provides users with a powerful [Admin API](./admin-api.md) and [AP
 
 We can create a [Route](./architecture-design/route.md) and connect it to an Upstream service(also known as the [Upstream](./architecture-design/upstream.md)). When a `Request` arrives at Apache APISIX, Apache APISIX knows which Upstream the request should be forwarded to.
 
-Because we have configured matching rules for the Route object, Apache APISIX can forward the request to the corresponding Upstream service. The following code is an example of a Route configuration:
+Because we have configured matching rules for the Route object, Apache APISIX can forward the request to the corresponding Upstream service. The following code creates a sample configuration of Route:
 
 ```json
```
Comment (spacewander) [inline docs/en/latest/getting-started.md:143]:
```suggestion
```bash
```

File: docs/zh/latest/getting-started.md [hunk 2]
```diff
@@ -136,35 +136,36 @@ Apache APISIX 提供了强大的 [Admin API](./admin-api.md) 和 [Dashboard](htt
 
 我们可以创建一个 [Route](./architecture-design/route.md) 并与上游服务（通常也被称为 [Upstream](./architecture-design/upstream.md) 或后端服务）绑定，当一个 `请求（Request）` 到达 Apache APISIX 时，Apache APISIX 就会明白这个请求应该转发到哪个上游服务中。
 
-因为我们为 Route 对象配置了匹配规则，所以 Apache APISIX 可以将请求转发到对应的上游服务。以下代码是一个 Route 配置示例：
+因为我们为 Route 对象配置了匹配规则，所以 Apache APISIX 可以将请求转发到对应的上游服务。以下代码会创建一个示例 Route 配置：
 
 ```json
```
Comment (spacewander) [inline docs/zh/latest/getting-started.md:141]: Ditto

File: docs/en/latest/getting-started.md (hunk 1 above)
Comment (piglei) [reply docs/en/latest/getting-started.md]: Updated.

**Comment by juzhiyuan** [comment]
Also cc @yzeng25 to have a review 🥰

**Review by spacewander — APPROVED** [review]

**Review by yzeng25 — APPROVED** [review]
LGTM

**Review by juzhiyuan — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

**Pull request merged**